.circleci/config.yml

```diff
@@ -240,6 +240,17 @@ jobs:
       - run: git config --global user.email "core-team@gatsbyjs.com"
       - run: sh ./scripts/publish-starters.sh "starters/*"
 
+  sync-i18n-source:
```

Comment:
question about naming: I'm calling this script `update-source` so far since I'm going to make a `sync` script that updates the translation repos based on the source. Could we call this `update-i18n-source`?

Reply:
Absolutely! I just picked some random name and I'm not particularly good at naming anyway hehe 

I like `update-i18n-source` better as well